### DISCUSSION TITLE
【社交链接】能否增加一个google scholar 或者自定义的社交链接添加方式(e.g. 知乎，乃至其他社交媒体)

### DISCUSSION THREAD

**Discussion by LuZeking** [tangly1024/NotionNext#2060] [Ideas]
**为什么提出这个新的特性改动**
对不同的读者需要提供不同的社交媒体方式

**描述一下你推荐的解决方案**
增加更多自定义社交链接的方式

**描述一下你考虑过的其它替代解决方案**
不能自定义的话希望可以增加google scholar, youtube, b站， 知乎等主流媒体社交链接， 提前感谢！

**Comment by Xuyan-cmd** [comment]
这个可以自行添加的，例如，在hexo主题下，可以修改/themes/hexo/components/SocialButton.js文件内容，如
![屏幕截图 2023-05-15 162642](https://github.com/tangly1024/NotionNext/assets/74172512/f2f0707a-00ba-4a34-89e3-7279e01746d5)
然后再修改blog.config.js文件内容，添加上配置好的社交链接：
![Screenshot_1](https://github.com/tangly1024/NotionNext/assets/74172512/d474fa51-4fb9-4b51-b7b4-51b442f5a9aa)
最后附上我的博客效果：  [百川的博客](https://www.baichuanweb.cn/)

**Comment by tangly1024** [comment]
感谢 @Xuyan-cmd 的评论，由于精力有限，我尽可能地加上各种社交链接；

如果愿意的话，可以自行修改，并将此特性PR至仓库中，将成功共享给更多人，十分感谢！

**Comment by Xuyan-cmd** [comment]
好的，周末有空会进行整理
